extensions/wikia/WallNotifications/styles/WallNotifications.globalNavigation.scss

```diff
@@ -6,17 +6,22 @@
 @import 'extensions/wikia/Venus/styles/mixins';
 @import 'extensions/wikia/Venus/styles/typographyMixins';
 
-$header-bg-color: #092140;
+$black-half-transparent: rgba(0, 0, 0, .6);
+$black-transparent: rgba(0, 0, 0, 0);
 $border-color: #c7d1d8;
-$wiki-header-color: #2b54b5;
-$wiki-header-bg-color: rgba(113, 133, 148, .2);
+$header-bg-color: #092140;
+$markasread-bg-color: rgb(9, 35, 68);
+$markasread-color: #fff;
+$notification-bubble-color: rgb(196, 206, 217);
 $notification-hover: rgba(164, 182, 191, .2);
+$notifications-divider-color: rgb(197, 206, 217);
```

Comment:
this color is already defined in GlobalNavigationSearch.scss as search input border color, please move them to some shared space.